static/scss/comment.scss

```diff
@@ -121,12 +128,24 @@
       }
     }
 
-    .replies > .comment {
-      padding-left: 8px;
+    .replies > & {
```

Comment:
This change broke some of the layout, if you look at the profile images, they're now touching the vertical dividers:

Your branch:
![selection_106](https://user-images.githubusercontent.com/28598/33397108-9ca56126-d518-11e7-9b17-46dc73a75bc7.png)


Master:
![selection_105](https://user-images.githubusercontent.com/28598/33397089-8d3c2eae-d518-11e7-8967-8e9bd00825b0.png)